frontend/src/Components/Charts/ContrabandHitrate/ContrabandHitrate.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+import { useTheme } from 'styled-components';
+
+// Router
+import { useParams } from 'react-router-dom';
+
+// Util
+import toTitleCase from 'util/toTitleCase';
+
+// State
+import useDataset, { CONTRABAND_HIT_RATE } from 'hooks/useDataset';
+
+// Children
+import ChartBase from 'Components/Charts/ChartBase';
+import Select from 'Components/Elements/Inputs/Select';
+import Bar from '../ChartTypes/Bar';
+
+const CHART_TITLE = 'Contraband Hit-rate';
+
+// In most cases, the reverse of this order ends up on the screen
+const GROUP_KEYS = ['hispanic', 'other', 'asian', 'native_american', 'black', 'white'];
```

Comment:
Is this a consequence of something we can't control, or deliberate design decision?

Reply:
Something we can't control. The charting library ingests the array and iterates on it in such a way that it comes out reversed. It's easy enough to reverse it before going in if we need to.  This is more of a note that we can in fact control the order the keys appear.